html/story.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Story</title>
+    <link href="../css/style.css" rel="stylesheet">
+    <link href="../css/top-stories.css" rel="stylesheet">
+    <link href="../css/story.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com" rel="preconnect">
+    <link crossorigin href="https://fonts.gstatic.com" rel="preconnect">
+    <link href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="main-container">
+        <header class="header">
+            <div class="header-container">
+                <div class="header-logo">
+                    <span class="login-header-text-first header-text-logo">ELEKS</span>
+                    <span class="login-header-text-second">BLOG</span>
+                </div>
+                <div class="search">
+                    <img alt="#" src="../assets/search-logo.png">
+                    <input class="search-input" placeholder="Search blog" type="search">
+                </div>
+            </div>
+            <div class="profile">
+                <img alt="#" src="../assets/profile-icon.png">
+            </div>
+        </header>
+        <div class="bellow-header">
+            <h3 class="third-head-text">All stories</h3>
+        </div>
+
+        <div class="small-container">
+            <img alt="#" src="https://via.placeholder.com/638x236.png">
```

Comment:
Can we download that picture and use it from assets? Just in case someday that link won't work